src/main/java/com/google/sps/utils/data/DataCollection.java

```diff
@@ -109,15 +109,16 @@ private static void parseSpeciesTable(String url) throws IOException {
    */
   private static Species processSpecies(Elements tds, String url) {
     if (tds.size() > 6) {
-      String commonName = tds.get(0).text();
-      String binomialName = tds.get(1).text();
-      String population = scrapePopulation(tds.get(2).text());
-      String status = scrapeStatus(tds.get(3).text());
+      String commonName = tds.get(0).text().trim();
+      String binomialName = tds.get(1).text().trim();
+      String populationString = scrapePopulation(tds.get(2).text()).trim();
+      long population = convertPopulationLong(populationString);
+      String status = scrapeStatus(tds.get(3).text()).trim();
       PopulationTrend trend = scrapeTrend(tds.get(4).select("img").first());
-      String notes = tds.get(5).text();
+      String notes = tds.get(5).text().trim();
```

Comment:
Can you remove the brackets for notes as well? `removeBrackets()` should do it for you

Reply:
Done!